wasm2c/wasm-rt.h

```diff
@@ -116,7 +122,7 @@ extern "C" {
 #endif
 
 /** Current call stack depth. */
-extern uint32_t wasm_rt_call_stack_depth;
+extern WASM_RT_THREAD_LOCAL uint32_t wasm_rt_call_stack_depth;
```

Comment:
Does this variable need to be external/public?  Can this be moved to line 5 of `wasm2c.declarations.c` and declared as a `static` an internal name?

Since that is the only use of `WASM_RT_THREAD_LOCAL`  in the external/public header perhaps that means we can avoid defining `WASM_RT_THREAD_LOCAL` publicly?